tests/app.yaml

```diff
@@ -351,3 +351,26 @@ cp2k:
       result:
         regex:  'CP2K\s+1\s+1.0'
         action: store_true
+
+
+#-------------------------------------------------------------------
+# Unicycler Test
+#-------------------------------------------------------------------
+unicycler:
+  summary: Unicycler test
+  build:
+    cmds:
+      - wget https://github.com/rrwick/Unicycler/raw/main/sample_data/short_reads_1.fastq.gz
+      - wget https://github.com/rrwick/Unicycler/raw/main/sample_data/short_reads_2.fastq.gz
```

Comment:
This will download these files everytime the test is run. Create a tarball for the test that includes the data files and place it in the test_src directory. Then specify the name of the tarball in the source_path:. See other tests for examples.

Reply:
The files are only downloaded once, however I have have moved the downloaded
files to ./test_src/Unicycler/